app/src/main/res/layout/answers_table.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <ScrollView
+        android:layout_width="410dp"
```

Comment:
I don't think it's a good idea to hard code the sizes of widget if we want our app to be used on many devices.

Reply:
I agree, as I said before it was done automatically when I dragged it into the design, next time I'll be more careful